lib/constants/firebaseConfig.dart

```diff
@@ -292,4 +293,9 @@ class FirebaseConfig {
       '"title": "NIK Tidak Terdaftar",'
       ' "description": "<p>Hanya NIK yang terdaftar untuk melakukan karantina mandiri yang dapat mengakses menu ini</p><p>Apabila Anda memiliki pertanyaan atau saran mengenai Kebijakan Privasi kami, jangan ragu untuk hubungi kami melalui email</p>"'
       '}';
+
+  static String selfIsolationUrlDefaultValue = '{'
+      '"doctor_consultation_url": "https://wa.me/6285697391854?text=Halo%20Admin!%20Saya%20ingin%20tanya%20seputar%20Isolasi%20Mandiri",'
+      '"vitamin_application_url": "https://bit.ly/pcbr-isomanvit"'
```

Comment:
@jaluowen why not use remote config?

Reply:
already use it mas, itu buat ngisi default value aja kalo gagal ngambil dari remote config

Reply:
oh i see ini default value ya.